chpl/chpl-service/src/test/java/gov/healthit/chpl/web/controller/CertifiedProductControllerTest.java

```diff
@@ -441,6 +438,204 @@ public void test_updatePendingCertifiedProductDTO_icsAndRetiredTTs_warningvsErro
+		updateRequest.setCertificationEdition(certificationEdition);
+		updateRequest.setIcs(true); // Inherited Status = product.getIcs();
+		updateRequest.setChplProductNumber("15.07.07.2642.EIC04.36.0.1.160402");
+		try {
+			ListingUpdateRequest listingUpdateRequest = new ListingUpdateRequest();
+			listingUpdateRequest.setListing(updateRequest);
+			certifiedProductController.updateCertifiedProduct(listingUpdateRequest);
+		} catch (InvalidArgumentsException e) {
+			e.printStackTrace();
+		} 
+		catch (ValidationException e) {
+			assertNotNull(e);
+			Boolean hasError = false;
+			for(String error : e.getErrorMessages()){
+				if(error.equalsIgnoreCase("Certification 170.314 (g)(4) contains Privacy and Security Framework value 'Approach 1, Approach 2' which must match either 'Approach 1', 'Approach 2', or 'Approach 1;Approach 2'")){
+					hasError = true;
+				}
+			}
+			assertTrue(hasError);
+		}
+	}
+	
+	/** 
+	 * GIVEN A user edits a certified product
+	 * WHEN they view their pending products
+	 * THEN they should see errors if any privacy and security framework values do not match one of "Approach 1", "Approach 2", "Approach 1;Approach 2"
+	 * (Note: Validation should be generous with case and whitespace)
+	 * @throws IOException 
+	 * @throws ParseException 
+	 * @throws JSONException 
+	 */
+	@Transactional
+	@Rollback(true)
+	@Test
+	public void test_updatePendingCertifiedProductDTO_privacyAndSecurityFramework_badValueShowsError() throws EntityRetrievalException, EntityCreationException, IOException, ParseException {
+		SecurityContextHolder.getContext().setAuthentication(adminUser);
+		PendingCertifiedProductDTO pcpDTO = new PendingCertifiedProductDTO();
+		String certDateString = "11-09-2016";
+		DateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy");
+		Date inputDate = dateFormat.parse(certDateString);
+		pcpDTO.setCertificationDate(inputDate);
+		pcpDTO.setId(1L); // Certified_product_id = 1 has icsCode = true and is associated with TestTool with id=2 & id = 3 that have retired = true
+		List<CertificationResultTestTool> crttList = new ArrayList<CertificationResultTestTool>();
+		CertificationResultTestTool crtt = new CertificationResultTestTool();
+		crtt.setId(1L);
+		crtt.setRetired(true);
+		crtt.setTestToolId(2L);
+		crtt.setTestToolName("Transport Test Tool");
+		crttList.add(crtt);
+		List<PendingCertificationResultDTO> pcrDTOs = new ArrayList<PendingCertificationResultDTO>();
+		PendingCertificationResultDTO pcpCertResultDTO1 = new PendingCertificationResultDTO();
+		pcpCertResultDTO1.setPendingCertifiedProductId(1L);
+		pcpCertResultDTO1.setId(1L);
+		pcpCertResultDTO1.setAdditionalSoftware(null);
+		pcpCertResultDTO1.setApiDocumentation(null);
+		pcpCertResultDTO1.setG1Success(false);
+		pcpCertResultDTO1.setG2Success(false);
+		pcpCertResultDTO1.setGap(null);
+		pcpCertResultDTO1.setNumber("170.314 (g)(4)");
+		pcpCertResultDTO1.setPrivacySecurityFramework("Approach 1, Approach 2");
```

Comment:
Something like `Approach 1 and Approach 2` should count as an invalid value. We can't support everything, but semi-colons and commas should be interchangable